locales/cs.json

```diff
@@ -0,0 +1,16 @@
+{
+  "empty_slot": "Prázdný slot",
+  "create_character": "Vytvořit novou postavu",
+  "select_character_title": "Výběr Postavy",
+  "delete_character_title": "Smazat Postavu",
+  "select_character": "Vyber si svou postavu",
```

Comment:
Zvolit postavu

Reply:
Thank your for spotting these mistakes, I have fixed them, but I think "Vybrat postavu" sounds better than "Zvolit postavu" as you suggested.